### DISCUSSION TITLE
About changing LCD localization, garbled?

### DISCUSSION THREAD

**Discussion by hyi1233** [mriscoc/Ender3V2S1#104] [Q&A]
According to god's instructions, after the LCD display is modified to Chinese, garbled characters appear in the interface menu. What else should I do?

#define LCD_LANGUAGE zh_CN
![Uploading IMG_20220330_083323.jpg…]()
![IMG_20220330_083344](https://user-images.githubusercontent.com/62958220/160730912-3a38bb2c-60c1-4859-83e6-e22192a21fe4.jpg)

**Comment by mriscoc** [comment]
There are no Chinese chars available in the integrated font and current version of Profesional firmware.
For Chinese support you should use the DWIN_LCD_PROUI version from official Marlin repository.
